src/main/resources/view/RightPanel.fxml

```diff
@@ -17,7 +18,10 @@
                 <Label fx:id="ownYouLabel" text="You are owed $1550.50" styleClass="label, owes-you-text"/>
```

Comment:
typo `own`? Does the text `text="You are owed $1550.50"` gets displayed in any case?

Reply:
"You are owed $1550.50" is nvr displayed in any case, I will removed to avoid confusion.